ios/flutter_blue.podspec

```diff
@@ -23,7 +23,7 @@ Bluetooth Low Energy plugin for Flutter.
     ss.source_files = "gen/*.pbobjc.{h,m}", "gen/**/*.pbobjc.{h,m}"
     ss.header_mappings_dir = "gen"
     ss.requires_arc = false
-    ss.dependency "Protobuf", '~> 3.11.4'
+    ss.dependency "Protobuf", '~> 3.11'
```

Comment:
Is this a downgrade or an upgrade? After the version change did the generated protobuf files change?

Reply:
Here, I widen the allowed version range for `Protobuf` dependency. I did not want to introduce a breaking change if somebody else, for some reason, needs to use version `3.11` of `Protobuf`. 

Right now, I do not have a good way to test if it changed the generated protobuf files, unfortunately. I'm on a PC and the only environment with MacOS is our CI which started failing because some other dependency of mine now requires a newer version of `Protobuf`. This is all, I needed to change for it to work, so I just wanted to upstream it, because others might be facing similar issues in near future if other pods require newer and newer versions of `Protobuf`.

Reply:
Gotcha, there are a few other PRs which make the same change, so you are not alone.

Reply:
I'd add a description to you PR if you'd want it to be merged tho